portal/views/assessment_engine.py

```diff
@@ -616,11 +616,11 @@ def assessment(patient_id, instrument_id):
             for answer in question['answer']:
                 # Hack: Extensions should be a list, correct in-place if need be
                 # todo: migrate towards FHIR spec in persisted data
-                if (
-                    'extension' in answer.get('valueCoding', {}) and
-                    not isinstance(answer['valueCoding']['extension'], (tuple, list))
-                ):
-                    answer['valueCoding']['extension'] = [answer['valueCoding']['extension']]
+                if ('extension' in answer.get('valueCoding', {})
```

Comment:
Please leave each condition on the same indentation level if you can. ie don't have a condition on the same line as the `if`, if there are multiple conditions